packages/kasumah-logger/package.json

```diff
@@ -16,7 +16,7 @@
   },
   "scripts": {
     "build": "./scripts/release.sh",
-    "typechain": "typechain 'artifacts/!(build-info)/**/+([a-zA-Z0-9]).json' --target ethers-v5",
+    "typechain": "typechain '{artifacts/!(build-info)/**/+([a-zA-Z0-9]).json,./node_modules/@openzeppelin/contracts/build/contracts/ERC1155PresetMinterPauser.json}' --target ethers-v5",
```

Comment:
Why is this required?

Reply:
for the tests - just so we can deploy an erc1155 for them.